operators/pkg/controllers/labinstance_controller.go

```diff
@@ -22,19 +22,18 @@ import (
 	"strings"
 	"time"
 
-	"github.com/netgroup-polito/CrownLabs/operators/labInstance-operator/pkg/instanceCreation"
+	"github.com/netgroup-polito/CrownLabs/operators/pkg/instanceCreation"
 
 	"github.com/go-logr/logr"
 	"github.com/google/uuid"
-	instancev1alpha1 "github.com/netgroup-polito/CrownLabs/operators/labInstance-operator/api/v1alpha1"
-	virtv1 "github.com/netgroup-polito/CrownLabs/operators/labInstance-operator/kubeVirt/api/v1"
-	templatev1alpha1 "github.com/netgroup-polito/CrownLabs/operators/labInstance-operator/labTemplate/api/v1alpha1"
+	instancev1alpha1 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha1"
```

Comment:
```suggestion
	crownlabsv1alpha1 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha1"
```

Just a proposal though, if it easy to be done